examples/compiled/geo_repeat.vg.json

```diff
@@ -30,7 +28,7 @@
         {
           "type": "geojson",
           "geojson": "geo",
-          "signal": "child__repeat_row_population_geojson_0"
+          "signal": "__row_population_geojson_0"
```

Comment:
To minimize review churns, I'd rather have consistent naming pattern with the past. You can send another PR to change / simplify it later, but avoid it in this giant PR.